dsc/tests/dsc_osinfo.tests.ps1

```diff
@@ -18,4 +18,20 @@ Describe 'Tests for osinfo examples' {
         $LASTEXITCODE | Should -Be 0
         $out.results[0].result.inDesiredState | Should -Be $IsMacOS
     }
+
+    It 'Verify dsc home directory is added to PATH to find included resources' {
+        $oldPath = $env:PATH
+        try {
+            $exe_path = Resolve-Path "$PSScriptRoot/../../bin/debug"
```

Comment:
I don't think you should rely on it being in `debug` folder.  Would be better to get the path to `dsc` and `set-location` to the parent folder then resolve relative path to `dsc` to execute it

Reply:
Good point; updated.